typescript/cli/src/deploy/core.ts

```diff
@@ -83,5 +86,7 @@ export async function runCoreDeploy({
 
     // @TODO implement writeAgentConfig
   }
-  logBlue('Deployment is complete!');
+
+  logGreen('✅ Core contract deployments complete:\n');
+  log(indentYamlOrJson(yamlStringify(deployedAddresses, null, 2), 4));
```

Comment:
Can you confirm this won't double-print them for a dry-run? I vaguely recall the dry-run printing these already but I could be remembering that wrong